packages/skill-api/src/core/services/send-feedback-from-user.ts

```diff
@@ -0,0 +1,63 @@
+import {htmlToText} from 'html-to-text'
+import sanitizeHtml from 'sanitize-html'
+import {getEmoji} from '../../lib/get-feedback-emoji'
+import {SendFeedbackFromUserOptions} from '../types'
+import {sendPostmarkEmail} from '../../lib/postmark'
+import {OutgoingResponse} from '../index'
+
+export async function sendFeedbackFromUser({
```

Comment:
this is a service/api function that interfaces with the database and email server to handle sending feedback

One big todo is figuring out consistent use of the Prisma client so that we can avoid configuring it in every project and get consistent testability.